cf/commands/application/apps_test.go

```diff
@@ -148,5 +150,125 @@ var _ = Describe("list-apps command", func() {
 				))
 			})
 		})
+		Context("when space flag is provided", func() {
+			BeforeEach(func() {
+				space := models.Space{}
+				space.Name = "my-space"
+				space.Guid = "my-space-guid"
+				spaceRepo.Spaces = []models.Space{space}
+			})
+			It("lists apps in a table", func() {
+				app1Routes := []models.RouteSummary{
+					models.RouteSummary{
+						Host: "app1",
+						Domain: models.DomainFields{
+							Name: "cfapps.io",
+						},
+					},
+					models.RouteSummary{
+						Host: "app1",
+						Domain: models.DomainFields{
+							Name: "example.com",
+						},
+					}}
+				app2Routes := []models.RouteSummary{
+					models.RouteSummary{
+						Host:   "app2",
+						Domain: models.DomainFields{Name: "cfapps.io"},
+					}}
+				app := models.Application{}
```

Comment:
Convert the app creation to something more like:
```
app := models.Application{
                           Name: "Application-1",
                           State: "started",
                           RunningInstances: `,
...
}
```

This reads better than what you have here. Please do the same also below.